src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -117,207 +120,203 @@ interface IOptionSettlementEngine {
      */
     error AccessControlViolation(address accessor, address permissioned);
 
+    // Input Errors
```

Comment:
I'd prefer using a full section header, and have them be "Errors - X", "Errors - Y", etc., or another type of section header. This // comment is too small and not idiomatic